### ISSUE TITLE
Week 1 Day 3 - Wireframes, Layouts, and Grid Systems

### ISSUE THREAD

**Issue by microsarah** [microsarah/WDI#13]
https://github.com/microsarah/WDI/tree/master/curriculum/week-01/03-wireframes-layouts-grid-systems

Ready for review @amyealmeida @carmenejames. 

Known issues:
- Needs SME input
- Hungry for More section is pretty bare. Essentially grids are a sister-lesson of Flexbox so we may not have a ton of resources here

**Comment by amyealmeida** [comment]
@microsarah I started reviewing this one, but was a little confused where you wanted the SME input to start and end.  Can you add comments for "SME Change to Flexbox: Start" "SME Change to Flexbox: End" to make it more clear which parts of the content need to be replaced?

Thank you!

**Comment by microsarah** [comment]
I decided to keep this lesson close the original. Making a grid with floats is similar to making a grid with flex, in that you still need to measure out widths in the same way. A flex grid is somewhat different, but we cover flexbox in a different lesson so I think what we have here is solid. Made sure the final code was added for students to download. 

Ready to close upon final review @amyealmeida ! :)
